### PR TITLE
create filter component

### LINKED ISSUE #84: Create Filter Component
image

- Create a Filter component that will take in the topic from props

### PR DESCRIPTION
**Pull request by pondohoo**
I'm still fairly new to web dev, so let me know of any mistakes I made. Also I wasn't sure if it needed to be a button or not, so I didn't make it one

### CODE REVIEW

**Comment by pondohoo** [comment]
This is what it looks like when I run it
Screen Shot 2022-08-16 at 10 37 02

File: components/Forms/Resume.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
+import React from "react";
```
Comment (shahdivyank) [inline components/Forms/Resume.js]: Can you remove this file? I think this is from last time. You can remove any files/folders regarding forms

File: pages/forms/resume.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
```
Comment (shahdivyank) [inline pages/forms/resume.js]: This file can also be removed

File: components/Filter.js [hunk 3]
```diff
@@ -1,7 +1,14 @@
 import React from "react";
 
-const Filter = () => {
-	return <div>Filter</div>;
+const Filter = ({ topic }) => {
+	return (
+		<div className='w-[270px] h-[55px] flex justify-between bg-black rounded-full'>
```
Comment (shahdivyank) [inline components/Filter.js]: Avoid hard coded pixel values, it causes undefined and unpredictable behavior with different screen sizes. instead use percentages

File: components/Filter.js [hunk 4]
```diff
@@ -1,7 +1,14 @@
...
+			<div className='text-white w-9 h-9 self-center ml-5 bg-[#ff5370] rounded-full'></div>
```
Comment (shahdivyank) [inline components/Filter.js]: Avoid `w-9`, `ml-5` and `h-9` as they also cause unpredictable behavior with various screen sizes, use percentages instead

File: components/Filter.js [hunk 5]
```diff
@@ -1,7 +1,14 @@
...
+			<div className='font-lexend text-white self-center mr-5'>
```
Comment (shahdivyank) [inline components/Filter.js]: Dont use `mr-5`

File: components/Filter.js [hunk 6]
```diff
@@ -1,7 +1,14 @@
 import React from "react";
 
-const Filter = () => {
-	return <div>Filter</div>;
+const Filter = ({ topic }) => {
```
Comment (shahdivyank) [inline components/Filter.js]: Add a color option, since we want them to have different colors and make sure to apply that color as well

**Review by shahdivyank — CHANGES_REQUESTED** [review]

**Comment by shahdivyank** [comment]
Great start! Few changes mostly regarding responsiveness! I added some implementation in the Calendar.js file (you shouldn't need to edit it) that will give you an idea of how much space you will be given, this space is dynamic and will change depending on screen size. So you can use that as a way to test the size of each one. If you think you need more space than 50%, lemme know and we can look into it!

**Review by shahdivyank — APPROVED** [review]
YESSIR!!!! 🎉🎉

**Review by skyguy-netizen — APPROVED** [review]

**Pull request merged**